src/chrome/content/rules/SaltStack.com.xml

```diff
@@ -11,9 +11,13 @@
 -->
 <ruleset name="SaltStack.com">
 	<target host="saltstack.com" />
+	<target host="bootstrap.saltstack.com" />
 	<target host="docs.saltstack.com" />
+	<target host="repo.saltstack.com" />
 	<target host="www.saltstack.com" />
 
+	<rule from="^http://saltstack\.com/"
```

Comment:
This doesn't appear to be necessary, as HTTPS appears to function correctly on `saltstack.com`.
An explicitly defined redirect rule is usually used when the HTTP version redirects, but the HTTPS version has an invalid certificate or some other error.

Reply:
I work at saltstack and we are trying to move to an s3 bucket to do the redirection which doesn't have https on saltstack.com.  I was fixing this in advance.

Reply:
Oops, didn't expect that to be the case. Sorry!